### ISSUE TITLE
Можно ли использовать как  children у Marker Text и Image одновременно?

### ISSUE THREAD

**Issue by nikitabudkovskiy** [volga-volga/react-native-yamap#67]
Можно ли использовать как children у Marker Text и Image одновременно?

**Comment by ownikss** [comment]
В целом можно. Какие то проблемы?

**Comment by nikitabudkovskiy** [comment]
> В целом можно. Какие то проблемы?

Да, рендерится только первый элемент, который помещается как children

**Comment by ownikss** [comment]
Лучше показать пример кода

Если я правильно понял сейчас
```
<Marker>
  <Image/>
  <Text/>
</Marker>
```

Это можно заменить на 
```
<Marker>
  <View>
    <Image/>
    <Text/>
  </View>
</Marker>
```

**Comment by mvpkx** [comment]
> Это можно заменить на
> 
> ```
> <Marker>
>   <View>
>     <Image/>
>     <Text/>
>   </View>
> </Marker>
> ```

Столкнулся с такой же проблемой. При оборачивании нескольких элементов в View, рендерится только первый из обёрнутых :(
